packages/pluggableWidgets/datagrid-web/CHANGELOM.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+All notable changes to this widget will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## 1.2.0 - Unreleased
+
+### Added
+- We've implemented lazy filtering and sorting. Now the data grid v2 will not load all the data if you have sorting or filtering enabled.
+
+### Changed
+- We've fixed a problem while having text box widget inside a column with an on leave/change event it was loosing focus after trigger the events
+- We've fixed a problem with 'Configuration' attribute and its on change event that was causing too many changes in the database while a column was being resized.
+
+### Removed
+- We've removed 'react-table' library from our code, so now we do not rely on any 3rd party library anymore.
```

Comment:
This isn't relevant for endusers. I would make it more generic, like this:
```
### Changed
- We've made drastic code improvements that improve performance.
```

Reply:
I've just noticed I named the file wrongly.. so i will fix all together

Reply:
I agree with Isa that it's not relevant to mention how we mention our dependencies, but do we have prove that the performance is actually improved? As far as we know we just removed _a lot_ of LOC, but does it have a lower footprint now/is it quicker?

I would just mention that we fixed the original issue, as that was the goal. So something like 
```
### Changed

- Prevent onchange action being fired continuously while resizing a column, causing performance issues.
- Prevent the config from being overwritten when changing to a different config.
```

These are just suggestions btw

Reply:
Done